backend/de.metas.util.web/src/main/java/de/metas/util/web/filter/ApiAuditFilter.java

```diff
@@ -105,10 +118,31 @@ public void doFilter(final ServletRequest request, final ServletResponse respons
 		{
 			logger.error(t.getLocalizedMessage(), t);
```

Comment:
```
logger.error("", t);
```

Reply:
@teosarca why log `"""` instead of the localized message?